apis/metal3.io/v1alpha1/baremetalhost_webhook.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 // log is for logging in this package.
-var baremetalhostlog = logf.Log.WithName("baremetalhost-resource")
+var baremetalhostlog = logf.Log.WithName("webhooks.BareMetalHost")
```

Comment:
Prefer `Log.WithName("webhooks").WithName("BareMetalHost")` for consistency with e.g. https://github.com/metal3-io/baremetal-operator/blob/main/main.go#L243